jscomp/stdlib/camlinternalLazy.js

```diff
@@ -21,7 +20,7 @@ function force_lazy_block(blk) {
   try {
     var result = closure(/* () */0);
     blk[0] = result;
-    Caml_obj_runtime.caml_obj_set_tag(blk, Obj.forward_tag);
+    Caml_obj_runtime.caml_obj_set_tag(blk, 250);
```

Comment:
1. should generate comment
2. should have an option to disable cross module optimization